packages/lib-classifier/src/store/SubjectStore.js

```diff
@@ -59,7 +59,14 @@ const SubjectStore = types
       const classificationDisposer = autorun(() => {
         onPatch(getRoot(self), (patch) => {
           const { path, value } = patch
-          if (path === '/classifications/loadingState' && value === 'posting') self.advance()
+          if (path === '/classifications/loadingState' && value === 'posting') {
+            const feedback = getRoot(self).feedback
+            if (feedback.isActive && feedback.messages.length > 0) {
+              feedback.showFeedback()
```

Comment:
It'd be nice if we could use a pattern where feedback interrupts the action in progress, then resumes it. At the moment, it just blocks the subject queue from advancing and we have to remember to advance it again somewhere else in the code.

Reply:
This is still a problem. We have to remember to call `subjects.advance()` from the `hideFeedback` action.